src/popup/router/pages/AccountDetails.vue

```diff
@@ -222,7 +222,7 @@ export default {
     gap: 16px;
     padding: 4px;
     border-radius: 14px;
-    background-color: variables.$color-bg-3;
+    background-color: var(--screen-bg-color);
```

Comment:
This is actually wrong, because the tabs background should be darker than the screen background.

Reply:
i only replaced the backgrounds that have the same variables bg-3